pkg/machine/vmconfigs/config.go

```diff
@@ -52,6 +52,8 @@ type MachineConfig struct {
 
 	// Starting is defined as "on" but not fully booted
 	Starting bool
+
+	Rosetta  bool
```

Comment:
This is the wrong place to store that, applehv specific settings must go into the AppleHypervisor struct.

Reply:
thoiugh again, i think this should go to containers common.

Reply:
When I read the code after refactoring to v5 it was too difficult for me to push into the AppleHypervisor struct.
This is an ongoing consideration,  containers common is also a very good idea.

Reply:
fixed